content/en/docs/instrumentation/erlang/getting-started/phoenix.md

```diff
@@ -0,0 +1,271 @@
+file describes the `span` record structure, and explains what all the different
+fields are.
+
+```shell
+*SPANS FOR DEBUG*
+{span,64480120921600870463539706779905870846,11592009751350035697,[],
+      undefined,<<"/">>,server,-576460731933544855,-576460731890088522,
+      {attributes,128,infinity,0,
+                  #{'http.status_code' => 200,
+                    'http.client_ip' => <<"127.0.0.1">>,
+                    'http.flavor' => '1.1','http.method' => <<"GET">>,
+                    'http.scheme' => <<"http">>,'http.target' => <<"/">>,
+                    'http.user_agent' =>
+                        <<"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36">>,
+                    'net.transport' => 'IP.TCP',
+                    'net.host.name' => <<"localhost">>,
+                    'net.host.port' => 4000,'net.peer.port' => 62839,
+                    'net.sock.host.addr' => <<"127.0.0.1">>,
+                    'net.sock.peer.addr' => <<"127.0.0.1">>,
+                    'http.route' => <<"/">>,'phoenix.action' => home,
+                    'phoenix.plug' =>
+                        'Elixir.OtelPhxSetupWeb.PageController'}},
+      {events,128,128,infinity,0,[]},
+      {links,128,128,infinity,0,[]},
+      undefined,1,false,
+      {instrumentation_scope,<<"opentelemetry_phoenix">>,<<"1.1.0">>,
+                             undefined}}
+```
+
+These are the raw structured logs that will get sent when you configure the
+exporter for your preferred service.
+
+### Rolling The Dice
+
+Now we'll add an API endpoint that will let us roll the dice and return a random
+number between 1 and 6.
+
+To start, we'll add a route to the `/api` scope in your router
+
+```elixir
+# lib/your_app_name_web/router.ex
+scope "/api", OtelPhxSetupWeb do
+  pipe_through :api
+
+  get "/rolldice", DiceController, :roll
+end
+```
+
+Then we'll create a new file in the controllers folder for that module. We told
+the router that we will define a roll function, so we'll do that. It will return
+a `200` response code and the result of a `dice_roll` function, which we will
+emit a span for. We also want to set the value of the generated roll as an
+attribute on the span.
+
+```elixir
+# lib/your_app_name_web/controllers/dice_controller.ex
+defmodule YourAppNameWeb.DiceController do
+  use YourAppNameWeb, :controller
+  require OpenTelemetry.Tracer, as: Tracer
+
+  def roll(conn, _params) do
+    send_resp(conn, 200, dice_roll())
+  end
+
+  defp dice_roll do
+    Tracer.with_span("dice_roll") do
+      roll = Enum.random(1..6)
+
+      Tracer.set_attribute(:roll, roll)
+
+      to_string(roll)
+    end
+  end
+end
+
+```
+
+If you point your browser/curl/etc. to `http://localhost:4000/api/rolldice` you
+should get a random number in response, and a few spans in your console. I'll
+point out some of the most relevant bits.
```

Comment:
Should give some details about how this is showing the span name and attributes for 2 spans.